package.nls.json

```
@@ -12,6 +12,7 @@
 	"hexEditor.openFile": "Open Active File in Hex Editor",
 	"hexEditor.goToOffset": "Go To Offset",
 	"hexEditor.selectBetweenOffsets": "Select Between Offsets",
+	"hexEditor.copyAs": "Copy As",
```

```suggestion
	"hexEditor.copyAs": "Copy As...",
```
Just a small detail to let the user know there will be another action, like in "Open With...".